0.17/files/docker-entrypoint.sh

```diff
@@ -32,7 +32,7 @@ fi
 NRTMPSAVES=$( find -L "$SAVES" -iname \*.tmp.zip -mindepth 1 | wc -l )
 if [ $NRTMPSAVES -gt 0 ]; then
   # Delete incomplete saves (such as after a forced exit)
-  rm -f "$SAVES/*.tmp.zip"
+  rm -f $SAVES/*.tmp.zip
```

Comment:
```suggestion
  rm -f "$SAVES"/*.tmp.zip
```

The quotes need to be around the variable in case its value contains any spaces. The important thing is that the `*` is not within quotes.